packages/x-charts/src/BarChart/BarChart.test.tsx

```diff
@@ -0,0 +1,30 @@
+import * as React from 'react';
+import { createRenderer, describeConformance } from '@mui/internal-test-utils';
+import { BarChart } from './BarChart';
```

Comment:
It's a public API, it should be:

```suggestion
import { BarChart } from '@mui/x-charts/BarChart';
```

to get a bit closer to an end-to-end test, and also make it clearer it's public.

x the other cases.